atomic_reactor/plugins/prepub_flatpak_create_oci.py

```diff
@@ -93,6 +96,12 @@ def _export_filesystem(self):
             self.log.info("Cleaning up docker container")
             self.tasker.remove_container(container_id)
 
+    def _get_oci_image_id(self, oci_path):
+        cmd = ['skopeo', 'inspect', '--raw', 'oci:{}'.format(oci_path)]
+        raw_manifest = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        oci_image_manifest = json.loads(raw_manifest)
```

Comment:
It wasn't obvious to me initially that this was encoding safe for both python2 and python3, but it is - the output of check_output is a str/bytes, and json.loads() handles that with a default-encoding of UTF-8 in both cases.

It wouldn't be hard to do this without skopeo: you have to load the <oci_path>/index.json and find the manifest id in there, then find that under blobs, but certainly using skopeo simpifies, and since we're already depending on skopeo for pushing, the extra dependency isn't really a problem.

Reply:
The reason I went with skopeo here instead of checking the oci_path contents was that `index.json` is an OCI image index (like a docker manifest list). It could have multiple entries and it does not have a `platform` key I could query to fetch the right image manifest. Since I lack understanding on the flatpak side (could more entries be added to the index for some reason?), I decided to go with skopeo.